juniper_official/Solutions/Traffic-Blackhole/system-alarms.rule

```diff
@@ -0,0 +1,126 @@
+/*
+	* REQ: COL-1109
+	* Rule to collect System Alarms for Blackhole Detection RCA.
+*/
+healthbot {
+	topic hardware.fpc {
+		rule system-alarms {
+			keys [ resource id ];
+			synopsis "System Alarms";
+			description "System Alarms";
+			sensor sensor-system-alarms {
```

Comment:
Aren't device alarms already collected by EMS?

Reply:
Removed this rule as it is already collected.